lib/Agrammon/Model.pm6

```diff
@@ -147,6 +147,8 @@ class Agrammon::Model {
     has Agrammon::Model::Module @.evaluation-order;
     has Agrammon::Model::Module @.load-order;
     has ModuleRunner $!entry-point;
+    has %.filter-groups;
+    has %.module-hash;
```

Comment:
This does not seem to be used anywhere.

Reply:
- removed `module-hash` (currently not used anymore)
- `filter-groups` is used in `Agrammon/OutputFormatter/JSON.pm6`

Reply:
Yes, was only talking about `module-hash`.